### DISCUSSION TITLE
add some more examples

### DISCUSSION THREAD

**Discussion by holgerlembke** [vroland/epdiy#327] [General]
hi.

i gave up on building a pcb myself, so I finally bought one. currently recovering from a blindness caused by the bright green led on the board, I start to toy around with it and epdiy.h in Arduino IDE 2.x

I think I will add some structual work to the lib and make it a little bit more "arduino ide" style. And add some simple examples.

any opinions?

(it will be slow work...)

**Comment by martinberlin** [comment]
Hi Holger, you can just solder a RES or even remove the one that feeds the LED, then you won't have any more Green disturbance. 
All examples should work in Arduino too. 
We are thinking that maybe it's a good idea in the future to separate the Component from the examples. 
This component is also primarly written in C for Espressif IDF, that way it should also run in Arduino IDE, but currently the fastest is to use ESP-IDF since in Arduino IDE you cannot select:

- Experimental settings -> PSRAM to 120 Mhz
- Esp-system: Cache line size to 64 bytes

Without this the library will reduce the clock to adapt to this constraints hence having a slower driver in Arduino IDE. Still usable though, it won't be a problem, but is worth mentioning.

**Comment by holgerlembke** [comment]
I know.

But current setup feels so Un-Arduino that I fear it might scare away new users.

**Comment by martinberlin** [comment]
Try also the www-image JPEG download so you can preview how it renders an image in 16 grays

**Comment by holgerlembke** [comment]
Work in Progress:

https://github.com/holgerlembke/epdiy/tree/main/examples/Basics
